t/nextjs_react_template/contents/src/app/auth.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+
+import { useUser } from '@auth0/nextjs-auth0/client';
+import { Avatar, Box, Button, IconButton, Link, Menu, MenuItem } from '@mui/material';
+import { AUTH_LOADING_TEXT, AUTH_LOGIN_TEXT, AUTH_LOGOUT_TEXT } from './constants';
+import { useState } from 'react';
+
+export default function Auth() {
+  const { user, error, isLoading } = useUser();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  if (isLoading) return <Box>{AUTH_LOADING_TEXT}</Box>;
```

Comment:
if () {}, we recommend wrapping your code block with curly braces {} every time you use an if statement